internal/pkg/workspace/errors.go

```diff
@@ -35,3 +35,13 @@ type ErrWorkspaceHasExistingProject struct {
 func (e *ErrWorkspaceHasExistingProject) Error() string {
 	return fmt.Sprintf("this workspace is already registered with project %s", e.ExistingProjectName)
 }
+
+// ErrAddonsDirNotExist means we tried to access in an addons dir for an application
+// but it does not exist.
+type ErrAddonsDirNotExist struct {
```

Comment:
Can we add the `Is` method to the error: https://blog.golang.org/go1.13-errors#TOC_4.

It allows us to do nicer error checks :)

Reply:
:+1: